cra/vulnerability-remediation.yaml

```diff
@@ -0,0 +1,308 @@
+---
```

Comment:
this file should be named `task-vulnerability-remediation.yaml` to be consistent with other files